feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/common/TotalAndTransferableBalance.kt

```diff
@@ -0,0 +1,33 @@
+package io.novafoundation.nova.feature_assets.domain.common
+
+import java.math.BigDecimal
+
+class TotalAndTransferableBalance(
```

Comment:
`XAndY` kind of naming is usually hard to understand later on since it is long. Can we rename to something else?